scripts/mediaViews.php

```diff
@@ -11,19 +11,25 @@
 $db = Doctrine_Manager::getInstance()->getCurrentConnection();
 
 //prune logs
-$date = date('Y-m-d H:i:s', strtotime('1 month ago'));
-
-$sql = "DELETE FROM media_views WHERE datecreated < ?";
+$sql = "DELETE FROM media_views WHERE datecreated < (NOW() - INTERVAL 1 MONTH)";
```

Comment:
It might be wiser to just save these views, in case we want to change the algorithm again later? Perhaps I'm misunderstanding how this works though, it looks like it was deleting them before...